### DISCUSSION TITLE
Theme question - text background color

### DISCUSSION THREAD

**Discussion by jem-os** [helix-editor/helix#11250] [Q&A] [answered]
What is the field name for text background color used in a theme please?

I've made my background color transparent using:
"ui.background" = {}

However, this isn't controlling the text background color.

**Comment by kirawi** [comment, marked as answer]
`ui.text`

If this is about strange backgrounds on text w/ syntax highlighting, see https://github.com/helix-editor/helix/wiki/Troubleshooting#the-theme-doesnt-look-as-it-should

**Reply by jem-os** [reply]
no, it's not about that. In Linux, you used to be able to take any standard theme and make the background transparent by creating a theme file in ~/.config/helix/insert-name-here.toml that contained two lines:

`inherits = "name-of-base-theme"`
`"ui.background" = {}`

this would keep the basics of the theme and just set the background for the document body to transparent, including text backgrounds. I typically used tokyonight_storm, and name my altered theme to toykonight_storm_transparent. However, this no longer works.

What it does now is set the background of the body where no text is present to transparent, but any text in the body has a background color equivalent to the bg color of the standard theme.

base16_transparent doesn't do this. It works like I expect. The background is transparent everywhere. Thus, I presume it's not an issue with a missing package, but with some parameter I'm not setting in my theme hack process.

It''s not clear to me, pun intended, what in the base16_transaparent.toml file is actually setting the background to transparent. For example, it has:
`"ui.background" = { fg = "white"}`

I was hoping for some guidance, as my standing presumption is that I'm doing something wrong.

Setting "ui.text" = {} solved the issue
